tests/test_replace_func.py

```diff
@@ -0,0 +1,174 @@
+import os
+import warnings
+
+import chainer
+import chainer.functions as F
+from chainer import testing
+import onnx
+import pytest
+
+from onnx_chainer import export_testcase
+from onnx_chainer import onnx_helper
+from onnx_chainer.replace_func import as_funcnode
+from onnx_chainer.replace_func import fake_as_funcnode
+from onnx_chainer.testing import input_generator
+from tests.helper import ONNXModelTest
+
+
+def test_fake_as_funcnode_without_replace(tmpdir):
+    path = str(tmpdir)
+
+    class Model(chainer.Chain):
+        def _init__(self):
+            super().__init__()
+
+        def add(self, xs, value=0.01):
+            return xs.array + value
+
+        def __call__(self, xs):
+            return F.sigmoid(self.add(xs))
+
+    model = Model()
+    x = input_generator.increasing(3, 4)
+
+    with warnings.catch_warnings(record=True):
+        export_testcase(model, x, path)
+
+    model_filepath = os.path.join(path, 'model.onnx')
+    assert os.path.isfile(model_filepath)
+
+    onnx_model = onnx.load(model_filepath)
+    node_names = {n.name for n in onnx_model.graph.node}
+    # no node is generated
+    assert len(node_names) == 0
+
+
+@testing.parameterize(
+    {'func_kind': 'list', 'in_shape': (2, 3, 4), 'op_type': 'Add'},
+    {'func_kind': 'list_kwargs', 'in_shape': (2, 3, 4), 'op_type': 'Add'},
+    {'func_kind': 'var_with_dec', 'in_shape': (3, 4),
```

Comment:
It took a few ten seconds to disambiguate this `dec` suffix. How about `deco` (or no abbreviation: i.e., decorator)?

Reply:
fixed